src/ai/backend/client/cli/session/args.py

```diff
@@ -0,0 +1,154 @@
+import click
+
+
+def decorator_group(*decs):
+    def d(f):
+        for decorator in decs[::-1]:
+            f = decorator(f)
+        return f
+
+    return d
+
+
+def click_start_option():
```

Comment:
How about adding type hint of return value?
And I suggest to declare the options in one list as a constant.
like the below.
```
START_OPTION = [
    click.option(),
    click.option(),
    ...
]
```